src/test/java/eu/europeana/metis/sandbox/service/workflow/OrderingServiceImplTest.java

```diff
@@ -34,8 +36,8 @@
 
   @Test
   void performOrdering_expectSuccess() throws IOException, TransformationException {
-    var input = testUtils.readFileToString("record/bad-order/record-input.xml");
-    var expected = testUtils.readFileToString("record/bad-order/record-expected.xml");
+    var input = testUtils.readFileToString("record"+File.separator+"bad-order"+File.separator+"record-input.xml");
+    var expected = testUtils.readFileToString("record"+File.separator+"bad-order"+File.separator+"record-expected.xml");
```

Comment:
Could you use the google stylesheet formatting in all of these classes ? (unless you already did)

Reply:
Done